Backpack/Card/Classes/BPKCardDivider.h

```diff
@@ -0,0 +1,66 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#import <UIKit/UIKit.h>
+#import <Foundation/Foundation.h>
+#import "BPKCard.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+/**
+ * `BPKCardDivider` is a subclass of `UIView` which contains the Skyscanner card divider style.
+ */
+NS_SWIFT_NAME(CardDivider) IB_DESIGNABLE @interface BPKCardDivider: UIView
+
+/**
+ * The orientation of the card divider.
+ */
+@property (nonatomic, assign) IBInspectable UILayoutConstraintAxis orientation;
+
+/**
+ * Create a `BPKCardDivider` with orientation initially set.
+ *
+ * @param orientation Whether the divider is for a vertical or horizontal layout.
+ */
+- (instancetype)initWithOrientation:(UILayoutConstraintAxis)orientation NS_DESIGNATED_INITIALIZER;
+
+/**
+ * Create a `BPKCardDivider` with a given frame.
+ *
+ * @param frame The initial frame of the card.
+ * @return `BPKCardDivider` instance.
+ */
+- (instancetype)initWithFrame:(CGRect)frame NS_DESIGNATED_INITIALIZER;
```

Comment:
Given that we never expect this to be created from storyboard or by consumers could we just mark this and `initWithCoder:` as unavailable?